Assets/Scripts/P2PNetcodeSample/Networking/EOSTransportManager.cs

```diff
@@ -1349,6 +1396,59 @@ private void OnConnectionClosedNotification(ref OnRemoteConnectionClosedInfo dat
             CloseConnection(remoteUserId, socketName, true);
         }
 
+        private void OnConnectionEstablishedNotification(ref OnPeerConnectionEstablishedInfo data)
+        {
+            // Sanity check
+            Debug.Assert(data.LocalUserId == LocalUserId);
+
+            var socketName = data.SocketId?.SocketName;
+            var remoteUserId = data.RemoteUserId;
+
+            // If we're establishing a connection, but no incoming connection was made yet, open an incoming connection
+            // This will also result in creating the connection to use
+
+            bool shouldCreateIncomingConnection = false;
+            Connection connection = null;
+
+            if (!Connections.TryGetValue(remoteUserId, out List<Connection> foundConnections))
+            {
+                shouldCreateIncomingConnection = true;
+            }
+            else
+            {
+                connection = foundConnections.Find(x => x.SocketName == socketName);
```

Comment:
Note to self: Determine without a doubt that this is not a `using System.Linq` related function call.

Reply:
Confirmed; this is from `System.Collections.Generic.List`. `Find` is a part of the List definition.